Content/Data/Items-External.xml

```diff
@@ -529,4 +529,43 @@
     <source>(0,0,45,45)</source>
     <offset>(-10,-10)</offset>
   </item>
+<<<<<<< HEAD
+  <!--Basic Orc Body-->
+  <item id="5345" category="16">
+    <texture>Tenser/monsters/basic_orc_pk1</texture>
+    <source>(15,75,45,45)</source>
+  </item>
+  <!--Basic Armored Orc Body-->
+  <item id="5350" category="16">
+    <texture>Tenser/monsters/basic_armored_orc_dead2</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+  <!--Armored Orc Body-->
+  <item id="5362" category="16">
+    <texture>Tenser/monsters/armored_orc_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+  <!--Basic Orc Caster Body (red)-->
+  <item id="5368" category="16">
+    <texture>Tenser/monsters/redrobe_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+  <!--Orc Caster Body (Black)-->
+  <item id="5371" category="16">
+    <texture>Tenser/monsters/blackrobe_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+  <!--Orc Praetor Body (plain)-->
+  <item id="5374" category="16">
+    <texture>Tenser/monsters/praetorian_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+  <!--Orc Praetor Body (trimmed)-->
+  <item id="5377" category="16">
+    <texture>Tenser/monsters/praetorian2_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+</data>
+=======
```

Comment:
Merge line endings are here. Need to clean those up and make sure the xml is valid.